### DISCUSSION TITLE
SyntaxError: Invalid or unexpected token from generated client/index.js

### DISCUSSION THREAD

**Discussion by iwaffles** [prisma/prisma#21273] [Q&A]
### Question

After running `prisma generate` and attempting to run the express server with node 20.6.0, I'm seeing the following error:

```
node_modules/.prisma/client/index.js:900
  }) ?? alternativePaths[0]
      ^

SyntaxError: Invalid or unexpected token
```

I'm using the nullish coalescing operator in many places in this project and have tried reinstalling prisma.

Is there another way around this or has anyone run into this issue before? 

### Expected behavior (optional)

Expect the generates client/index.js file to not have any syntax errors in this environment.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: Debian via docker container (dev container running in WSL 2.0)
- Database: PostgreSQL
- Node.js version: 20.6.0
- Run `prisma -v` to see your Prisma version and paste it
```
Environment variables loaded from .env
prisma                  : 5.3.1
@prisma/client          : 5.3.1
Current platform        : debian-openssl-1.1.x
Query Engine (Node-API) : libquery-engine 61e140623197a131c2a6189271ffee05a7aa9a59 (at node_modules/@prisma/engines/libquery_engine-debian-openssl-1.1.x.so.node)
Schema Engine           : schema-engine-cli 61e140623197a131c2a6189271ffee05a7aa9a59 (at node_modules/@prisma/engines/schema-engine-debian-openssl-1.1.x)
Schema Wasm             : @prisma/prisma-schema-wasm 5.3.1-2.61e140623197a131c2a6189271ffee05a7aa9a59
Default Engines Hash    : 61e140623197a131c2a6189271ffee05a7aa9a59
Studio                  : 0.494.0
```

**Comment by SonnyAzIgazi** [comment]
If you are using esm, that's the reason prisma fails. Solution:
[Link to comment](https://github.com/prisma/prisma/issues/18176#issuecomment-1453498422)
